src/stores/user.js

```diff
@@ -1,15 +1,14 @@
 import { defineStore } from 'pinia'
-import { supabase } from '../supabase'
 
 // eslint-disable-next-line import/prefer-default-export
 export const useUserStore = defineStore('user', {
   state: () => ({
     isSignedIn: false
   }),
   actions: {
-    async signUp(email, password) {
+    async authenticateUser(email, password, method) {
       try {
-        const { error } = await supabase.auth.signUp({
+        const { error } = await method({
```

Comment:
This won't work. When supabase methods are passed the lose context

Reply:
I understand, I change those functions back, then.